hkmc2/shared/src/main/scala/hkmc2/semantics/Symbol.scala

```diff
@@ -116,12 +116,16 @@ class BlockMemberSymbol(val nme: Str, val trees: Ls[Tree])(using State)
   override def toString: Str =
     s"member:$nme${State.dbgUid(uid)}"
 
+  override val isGetter: Bool = // TODO: should be based on another syntax & kind
```

Comment:
Can you clarify? I don't understand what this comment means.

Reply:
> should not define what is or isn't a getter based on this criterion, and rather use a special syntax.

Oh, I just left a TODO here for the future getter syntax.